contents/docs/error-tracking/stack-traces.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Stack traces
+---
+
+import BetaFeedbackBanner from './_snippets/beta-feedback-banner.mdx'
+
+<BetaFeedbackBanner />
+
+Error tracking provides you with the ability to view the stack trace and code context associated with an exception. This can help understand, identify and resolve the root cause of an issue.
+
+For languages like Python the stack trace and code context can be gathered clientside and requires no additional processing.
```

Comment:
What language is the example image in? Looks like server-side JavaScript?
```suggestion
For languages like Python, the stack trace and code context can be gathered by the PostHog client and requires no additional processing.
```

Reply:
It's actually frontend js with the React frames showing